src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -61,12 +68,28 @@
 
 <script>
 import _ from 'lodash';
+import moment from 'moment';
+
+const DEFAULT_DURATION_FORMATTER = 'duration';
+const AGE_TRACK_INTERVAL_MS = 100;
```

Comment:
Let's increase this. Tracking should happen no more often than every 1s.

Reply:
Done!